src/domains/fdr-common/03_storage_account_fdr.tf

```diff
@@ -141,45 +247,9 @@ resource "azurerm_storage_table" "fdr1_conversion_error_table" {
   storage_account_name = module.fdr_conversion_sa.name
 }
 
-## fdr 1 cached responses blob container
-resource "azurerm_storage_container" "fdr1_cached_response_blob_file" {
-  name                 = "fdr1-cached-response"
-  storage_account_name = module.fdr_conversion_sa.name
-}
-
 
 
 
-## 🐞https://github.com/hashicorp/terraform-provider-azurerm/pull/15832
```

Comment:
why remove it ?

Reply:
I thought it can be deleted because, due to high costs on transactions, it will be no more used and we could possibly gain some space on storage.  
I've done a preliminary analysis on this blob storage and I've found that there are lots of hidden files that uses too much memory (in the screenshot, a (<i>**partial**</i>) view of the content on this repository).  
But if it is required to not be deleted, I can revert this change, of course.
![Screenshot 2025-03-03 alle 12 51 43](https://github.com/user-attachments/assets/7b587787-1cf7-4d8c-bf0c-57ef607d14d1)